tests/slist_ut/slist_ut.c

```diff
@@ -0,0 +1,1114 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include "macro_utils/macro_utils.h"
+#include "umock_c/umock_c.h"
+#include "umock_c/umocktypes_bool.h"
+#include "umock_c/umocktypes_charptr.h"
+
+
+#include "c_util/slist.h"
+#include "c_util/containing_record.h"
+#include "testrunnerswitcher.h"
+
+typedef struct simpleItem_tag
```

Comment:
>simpleItem_tag [](http://example.com/codeflow?start=15&length=14)

Typically we use UPPER_CASE_STRUCT_TAG for structure names and tags

Reply:
done